docs/language-specification/statements/version-statement.md

```diff
@@ -0,0 +1,15 @@
+The version statement is mandatory and indicates which version of the cQASM language the quantum program is written in.
+Apart from comments, it must be the first statement of the program and occur only once.
+It has the following form,
+
+`version <M:INT>.<m:INT>`
+
+where the version number is to be given as two period separated integers, respectively indicating the major `M` and minor `m` version of the cQASM language.
+It is permitted to only specify the major version number `M`. In that case, the program is presumed to be written according to the specification of the latest minor `m` version.
```

Comment:
Is that the convention? I would rather have it select the latest version, otherwise omitting the minor version will have a very limited use. We can describe in the spec as it is implemented now, but shouldn't we change that?